charset/charset.go

```diff
@@ -50,7 +50,7 @@ func init() {
 func Reader(charset string, input io.Reader) (io.Reader, error) {
 	charset = strings.ToLower(charset)
 	// QUIRK: "ascii" and "utf8" are not in the spec but are common
```

Comment:
Can you update this comment?

Reply:
Definitely. Would it make sense to just include an entire list of aliases based on the [IANA registry](https://www.iana.org/assignments/character-sets/character-sets.xhtml) that was linked before?

Reply:
I believe most of these aliases are not in the actual use. Can we ignore them for now?